package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "build-clean": "rimraf ./packages/*/build ./packages/*/build-module",
     "build": "node ./scripts/build.js",
-    "postinstall": "lerna bootstrap --hoist && npm run build",
```

Comment:
Is removing `--hoist` intentional?

Reply:
Yes, I opened an issue to address it separately #56. I will rebase once it gets merged.